apps/web/src/common/components/header/styled.ts

```diff
@@ -0,0 +1,34 @@
+import styled from '@emotion/styled'
+
+export const HeaderContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 5px 66px;
+  position: absolute;
+  width: 100%;
+  height: 6vh;
+  left: 0px;
+  top: 0px;
+  background: #5b85aa;
```

Comment:
(Will be discussed how to style)
I'm not recommend to do this because, it's might lead to inconsistent theme when changing is needed